src/style/App.css

```diff
@@ -28,16 +36,76 @@ body {
   border-bottom: solid 1px var(--gray);
 }
 
+.nav-brand {
+  color: var(--skyblue);
+  font-weight: 700;
+  cursor: pointer;
+  text-decoration: none;
+}
+
+nav {
+  flex: 1;
```

Comment:
- [ ] Dear @yash244466666, As the previous code reviewer mentioned, adjust the background of the website to match the design guide. Notice the margin for the `Bookstore CMS`, `Navigations` and the `user icon` are in the same margin as the book cards. 

### Given design.
![216527567-63e86631-2b41-4a2d-a023-4ffc427fe05e](https://user-images.githubusercontent.com/69077061/216595669-734ba434-2dd7-4df9-af23-0f883c48a80f.png)

### Your design
![Screenshot 2023-02-03 144320](https://user-images.githubusercontent.com/69077061/216595672-fdf75c76-482c-4e1a-aa68-89613ad75ea5.png)